src/node/internal/buffer.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for c++ implementation
```

Comment:
I think it is actually possible to generate these for internal implementations too. I'll try to hack something up together.